src/index.ts

```diff
@@ -0,0 +1,80 @@
+import { TASK_COMPILE_GET_SOURCE_PATHS } from "@nomiclabs/buidler/builtin-tasks/task-names";
+import { task } from "@nomiclabs/buidler/config";
+import { BuidlerConfig } from "@nomiclabs/buidler/types";
+import { ensureDir, readFile, writeFile } from "fs-extra";
```

Comment:
This should be imported where needed. This file is loaded from the users' Buidler config, so keeping it fast is important.